reshape/fields/config.yaml

```diff
@@ -0,0 +1,12 @@
+access:
+  policies:
+    - type: Query
+      rules:
+        - condition: true
+          name: api
+          fields: [hominoid, person, defaultHero, luke, robot]
+        - condition: true
+          name: introspection
+          fields: [__schema, __type, __typename]
+      policyDefault:
```

Comment:
Some comments in this file will help understanding the mechanics of specifically defining the fields that are exposed would be in place. Especially, as users would expect all fields to be exposed by default